infra/cifuzz/stack_parser.py

```diff
@@ -45,12 +43,12 @@
 ]
 
 
-def parse_fuzzer_output(fuzzer_output, out_dir):
+def parse_fuzzer_output(fuzzer_output, file_path):
```

Comment:
s/file_path/output_file_path to make it more clear?

Reply:
Done.